pkg/nodes/node.go

```diff
@@ -23,10 +25,77 @@ const (
 
 // Builder provides struct for Node object containing connection to the cluster and the list of Node definitions.
 type Builder struct {
-	Definition *v1.Node
-	Object     *v1.Node
-	apiClient  *clients.Settings
-	errorMsg   string
+	Definition  *v1.Node
+	Object      *v1.Node
+	apiClient   *clients.Settings
+	errorMsg    string
+	drainHelper *drain.Helper
+}
+
+// NewDrainHelper builds drain Helper that contains parameters to control the behaviour of drain.
+func (builder *Builder) NewDrainHelper(
+	force bool,
```

Comment:
Please keep arguments of the same type close to each other.

Reply:
updated, thanks